script/install.sh

```diff
@@ -53,6 +53,9 @@ then
     if [ "${CXX}" == "clang++" ] ;then source ./script/install_clang.sh ;fi
 elif [ "$ALPAKA_CI_OS_NAME" = "macOS" ]
 then
+    echo "### list all applications ###"
+    ls "/Applications/"
+    echo "### end list all applications ###"
```

Comment:
Is this some kind of debug output? Do you still need it? Can it be removed?

Reply:
yes please keep it else we need to fire dummy CI tasks to get this information